### ISSUE TITLE
#3

### ISSUE THREAD

**Issue by kjell** [kjell/kombucha#3]
I bottled `8 x 16oz` two nights ago. I did one test the night before with lemon juice and ginger, and it turned out pretty well. I gave some to Eric W to test and he said it was good. I drank the rest of the bottle.

That left the big jar about 1/3 full. The tea still tasted OK, but it was getting pretty sour.

I brewed the new tea twice as strong, using `2 x 23g` tea satchets, and used `6 cups` of sugar instead of `4`.

I then mixed the hot sugary tea with enough cold filtered water in a big food safe plastic container to fill it up just right (lucky guess) and sat that in cold water in the sink until it was cool enough to dump in with the scoby. Pouring the fresh stuff in tumbled the SCOBY all around, but it floated pretty quickly back to the surface of the jar.

I'm going to keep this batch covered with an old pullover so it shouldn't see too much daylight. I guiltily turned the heat in my room up a few degrees to see how that affects the process, we'll see how long I can stand pushing 70ºF when I'm biking to work in -20º.

**Comment by kjell** [comment]
It's a lot fizzier so far. It tastes way too sugary but I think this round it's moving faster than it was 2 days in last batch. The scoby is now light brown across most of the surface of the jar, whereas most of the last batch it stayed clear.

**Comment by kjell** [comment]
The sugary taste is done, but this batch tastes a bit strong. I can drink a little but it turns my stomach if I drink more than a few sips. It is nice and fizzy.

Of the 8 bottles I did last batch only two built up any carbonation. The old plastic caps that I have need a piece of cardboard in the lid to seal it tight and it deteriorates fast. I picked up some new bottle caps today, and also some 16oz flip top (grolsch look-alike) bottles.

This round I'll do some lemon ginger and some hibiscus. I'm not a huge fan of plain, but I'll try one plain + sugar. Test a few bottles tonight and then the rest after.

**Comment by kjell** [comment]
I did 6 bottles 3 days ago. 3 flavors, and of each flavor I sugared one (with `1tsp` white sugar) and left the other unsweetened.
- 2 plain: boring. Neither built up much carbonation or flavor.
- 2 hibiscus: good. I put `1tbsp` of dried hibiscus flowers in each. Both fizzed quite when opened. I've only drank from the sweetened bottle, it was nice. I opened the other 5 minutes ago to test and it seems like it's just as bubbly.
- 2 lemon ginger: good. If I remember right each had `1/8 cup` of squeezed lemon juice and a few pinches of ground ginger. I was worried about the ginger because I ground it up (coffee grinder) a week earlier and it was looking brown and soggy from sitting in the fridge, but it tastes fine.

I've been pouring it into a glass and drinking with ice cubes.

Tonight I bottled 9 more. Right now I have cranberries, frozen orange peels and cardamom, so I'll see what I can do with those tonight and think up more flavors tomorrow.
